README.md

```diff
@@ -47,6 +47,7 @@ Additionally, it must fulfill the following requirements:
 
 - All code produced as part of a grant must be **open-sourced**, and it must also not rely on closed-source software for full functionality. We prefer Apache 2.0, but GPLv3, MIT or Unlicense are also acceptable.
 - We do not award grants for projects that have been the object of a successful token sale.
+- Applications must not mention a specific token. Furthermore, the focus of the application should lie on the software that is being implemented/research being carried out as part of the grant, and less on your project/venture/operation. For the purpose of the application, please think of your project as just one of many that may integrate your open-source implementation.
```

Comment:
I would remove: "For the purpose of the application, please think of your project as just one of many that may integrate your open-source implementation." Otherwise, it looks good.